docs/Inventory-of-Epics.md

```diff
@@ -0,0 +1,27 @@
+# "Inventory of Epics" for the Roslyn Analyzer Ecosystem
+
+Here is a current list of the major areas of work ("epics") for the Roslyn analyzer ecosystem. See the [README](https://github.com/dotnet/roslyn-analyzers-contrib/blob/master/README.md) for an explanation of what that means.
+
+1. Replace FxCop
```

Comment:
I'm pondering exactly what to say about this. In the meantime I'm going to merge this and close the issue; I think it's now had sufficient review.